Directory.Build.props

```diff
@@ -3,8 +3,15 @@
     <HighEntropyVA>true</HighEntropyVA>
     <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
     <TargetFramework>net6.0</TargetFramework>
-  </PropertyGroup>
-  <PropertyGroup>
+    
+    <Build_BuildNumber Condition="'$(Build_BuildNumber)' == ''">1.dev</Build_BuildNumber>
+    <AssemblyVersion>3.0.0.0</AssemblyVersion>
```

Comment:
> <AssemblyVers [](http://example.com/codeflow?start=3&length=14)

Shouldn't these version numbers be overridable by msbuild /p or by environment variables?

<MajorVersion Condition='$(MajorVersion)'==''>3</MajorVersion>
<AssemblyVersion>$(MajorVersion).$(MinorVersion).0.0</AssemblyVersion>

Reply:
Ideally coming from the pipeline... yes

Reply:
They can be overridden by pipeline variables. They're not as of now. I'm yet to discuss the vision of how we would like to maintain versioning scheme for longer run so we can design it that way, privately.